lib/puma/server.rb

```diff
@@ -293,8 +293,6 @@ def run(background=true)
         return run_lopez_mode(background)
       end
 
-      queue_requests = @queue_requests
```

Comment:
This removes a change from cc4ad1751977e55163ef583f05152288b4d23186, since we need to change behavior based on the updated value of the `@queue_requests` variable after the reactor is shutdown. I assume this was a performance optimization, but I don't know if it actually makes any measurable difference.